models/student.rb

```diff
@@ -1,11 +1,11 @@
 require_relative 'person'
+require_relative 'classroom'
 
 class Student < Person
-  attr_accessor :classroom
+  attr_accessor :age, :classroom, :name, :parent_permission
 
-  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
-    super(age, parent_permission, name)
-    @classroom = classroom
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    super(age: age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] Great job creating the `has-many` relationship between Classroom and Student 👏. Now, please create the `belongs-to` relationship from Student to Classroom as per requirement.
Kindly reference this [resource](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) for more clarification on how to do this 👍 

![image](https://github.com/SaneMyburg/school-library/assets/46839548/823b9099-d27d-4633-a85c-69aa7793f0a2)